src/orbit/lib/functions.js

```diff
@@ -12,4 +12,30 @@ var spread = function(func) {
   };
 };
 
-export { spread };
+function every(array, predicate) {
```

Comment:
@mitchlloyd `src/orbit/lib/arrays.js` might be a more appropriate home for every/some